src/components/accordion/accordion.stories.ts

```diff
@@ -264,28 +264,42 @@ export const withIconStartAndEnd_TestOnly = (): string =>
-    <calcite-accordion-item
-      heading="Accordion Item IconStart/End"
-      icon-start="plane"
-      icon-end="plane"
-      scale="l"
-      description="A great subtitle"
-    ></calcite-accordion-item>
-    <calcite-accordion-item
-      heading="Accordion Item IconStart/End with a potentially two line title"
-      icon-start="banana"
-      icon-end="banana"
-      scale="l"
-    ></calcite-accordion-item>
-  </calcite-accordion>
+const useCases = [
+  { icon: "", heading: "Simple item with heading", description: "" },
+  { icon: "", heading: "Simple item with heading", description: "Simple item with description" },
+  {
+    icon: "embark",
+    heading:
+      "Embark_title_is_super_long_what_do_we_do_now_Watercraft_title_is_super_long_what_do_we_do_now_Watercraft_title_is_super_long_what_do_we_do_now_Watercraft_title_is_super_long_what_do_we_do_now_Watercraft_title_is_super_long_what_do_we_do_now_Watercraft_title_is_super_long_what_do_we_do_now_Watercraft_title_is_super_long_what_do_we_do_now_Watercraft_title_is_super_long_what_do_we_do_now_Watercraft_title_is_super_long_what_do_we_do_now_Watercraft_title_is_super_long_what_do_we_do_now_Watercraft_title_is_super_long_what_do_we_do_now_",
+    description: "Extra long heading with underscores and icons m /scale l"
+  },
+  {
+    icon: "car",
+    heading: "Extra long description with underscores and icons m /scale l",
+    description:
+      "Car_title_is_super_long_what_do_we_do_now_Watercraft_title_is_super_long_what_do_we_do_now_Watercraft_title_is_super_long_what_do_we_do_now_Watercraft_title_is_super_long_what_do_we_do_now_Watercraft_title_is_super_long_what_do_we_do_now_Watercraft_title_is_super_long_what_do_we_do_now_Watercraft_title_is_super_long_what_do_we_do_now_Watercraft_title_is_super_long_what_do_we_do_now_Watercraft_title_is_super_long_what_do_we_do_now_Watercraft_title_is_super_long_what_do_we_do_now_Watercraft_title_is_super_long_what_do_we_do_now_"
+  },
+  {
+    icon: "plane",
+    heading: "Extra long description and icons m /scale l",
+    description:
+      "Planes, helicopters, and jets. Planes, helicopters, and jets. Planes, helicopters, and jets. Planes, helicopters, and jets. Planes, helicopters, and jets. Planes, helicopters, and jets. Planes, helicopters, and jets. Planes, helicopters, and jets. Planes, helicopters, and jets. Planes, helicopters, and jets. Planes, helicopters, and jets. Planes, helicopters, and jets. Planes, helicopters, and jets. Planes, helicopters, and jets. Planes, helicopters, and jets."
+  }
+];
+
+const accordionItems = useCases
+  .map(
+    (useCase): string =>
+      html`<calcite-accordion-item
+        icon-start="${useCase.icon}"
+        icon-end="${useCase.icon}"
+        scale="l"
+        heading="${useCase.heading}"
+        description="${useCase.description}"
+      ></calcite-accordion-item>`
+  )
+  .join("");
+
+export const longHeading_MediumIconForLargeAccordionItem_testOnly = (): string => html`
```

Comment:
Per [our guidelines](https://github.com/Esri/calcite-components/blob/3bd1b540aad7254bd34f726da926cc31e0897fc5/conventions/Documentation.md#screenshot-test-only-stories), suffix should be `_TestOnly`.